llvm/lib/Target/AArch64/AArch64InstrInfo.td

```diff
@@ -2145,6 +2145,12 @@ def MSR_FPSR : Pseudo<(outs), (ins GPR64:$val),
                PseudoInstExpansion<(MSR 0xda21, GPR64:$val)>,
                Sched<[WriteSys]>;
 
+let Uses = [FPMR], Defs = [FPMR, NZCV] in
```

Comment:
With the simpler usage can this be just `let Defs = [FPMR]`?

Reply:
Yes. forgot to remove this when switching to simple write. Fixed